### PR TITLE
Feature/issue 9 controller get users

### LINKED ISSUE #9: Issue 9 : Controller Get Users
**Priority :**
1

**Duration :** 
1

**Description:**
Creation of routes to list users

**File added / modified:**

**Additional context**

Add any other context or screenshots here.

### PR DESCRIPTION
**Pull request by vanmarcke**
Issue [#9]

Creation of routes to list users

### CODE REVIEW

File: src/Manager/UsersManager.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Manager;
+
+use App\Entity\Clients;
+use App\Repository\UsersRepository;
+use Doctrine\ORM\Query;
+
+class UsersManager implements UsersManagerInterface
```
Comment (antoinetrouve) [inline src/Manager/UsersManager.php]: `class UserManager implements UsersManagerInterface`

File: src/Manager/UsersManager.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+{
+    public function __construct(private UsersRepository $usersRepo)
+    {
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getUsersList(Clients $client): Query
```
Comment (antoinetrouve) [inline src/Manager/UsersManager.php]: `public function getUserList(Clients $client): Query`
OU
`public function getUsers(Clients $client): Query`

**Review by antoinetrouve — APPROVED** [review]

File: src/Manager/UsersManager.php (hunk 1 above)
Comment (vanmarcke) [reply src/Manager/UsersManager.php]: Done

File: src/Manager/UsersManager.php (hunk 2 above)
Comment (vanmarcke) [reply src/Manager/UsersManager.php]: done

**Pull request merged**